inc/Engine/Optimization/RUCSS/Frontend/APIClient.php

```diff
@@ -90,6 +90,9 @@ public function get_queue_job_status( $job_id, $queue_name, $is_home = false ) {
 		];
 
 		$result = json_decode( $this->response_body, true );
+		if ( key_exists( 'code', $result ) && 401 === $result['code'] ) {
+			set_transient( 'wp_rocket_no_licence', true, WEEK_IN_SECONDS );
```

Comment:
When we set the transient, we should also update the rucss option to set the value to zero